custom/auth/ante/tax_test.go

```diff
@@ -384,359 +384,3 @@ func (suite *AnteTestSuite) TestEnsureMempoolFeesExec() {
 	_, err = antehandler(suite.ctx, tx, false)
 	suite.Require().NoError(err, "Decorator should not have errored on fee higher than local gasPrice")
 }
-
-func (suite *AnteTestSuite) TestEnsureMempoolFeesSendLunaTax() {
```

Comment:
why do you remove a bunch of test here?

Reply:
Those tests were for old burn tax (100%) with TaxPowerUpgradeHeight

Reply:
I think we should update tests accordingly

Reply:
Okay. I will re-add TaxPowerUpgradeHeight then

Reply:
@nghuyenthevinh2000 Please resolve this thread it if you have no issue